lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -965,7 +970,9 @@ def run( self, trans, id, history_id=None, **kwargs ):
         workflow_contents_manager = workflows.WorkflowContentsManager( trans.app )
         stored = workflow_manager.get_stored_accessible_workflow( trans, id )
         workflow_dict = workflow_contents_manager.workflow_to_dict( trans, stored, style='run' )
-        return trans.fill_template( 'workflow/run.mako', workflow_dict=workflow_dict )
+        wf_dict = list()
+        wf_dict.append( workflow_dict )
+        return json.dumps( wf_dict )
```

Comment:
Should be just 'return workflow_dict'. Is there an api function which already does this?